avocado/core/dispatcher.py

```diff
@@ -83,6 +83,17 @@ def names(self):
     def _init_plugins(self, extensions):
         super(Dispatcher, self)._init_plugins(extensions)
         self.extensions.sort(key=lambda x: x.name)
+        configured_order = settings.get_value(self.settings_section(), "order",
+                                              key_type=list, default=[])
+        ordered = []
+        for name in configured_order:
+            for ext in self.extensions:
+                if name == ext.name:
+                    ordered.append(ext)
```

Comment:
Btw do you expect multiple plugins with the same name to be initialized in `self.extensions`? If not please use `break` here.